README.md

```diff
@@ -69,7 +69,20 @@ Notes:
 5. See [documentation](https://godoc.org/gopkg.in/reform.v1) how to use it. Simple example:
 
     ```go
-	// Use reform.NewDB to create DB.
+	// Get *sql.DB as usual. PostgreSQL example:
+	sqlDB, err := sql.Open("postgres", "postgres://127.0.0.1:5432/database")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer sqlDB.Close()
+
+	// Use new *log.Logger for logging.
+	logger := log.New(os.Stderr, "SQL: ", log.Flags())
+
+	// Create *reform.DB instance with simple logger.
+	// Any Printf-like function (fmt.Printf, log.Printf, testing.T.Logf, etc) can be used with NewPrintfLogger.
+	// Change dialect for other databases.
+	DB = reform.NewDB(sqlDB, postgresql.Dialect, reform.NewPrintfLogger(logger.Printf))
```

Comment:
`DB` in the upper case looks a bit confusing... but on the other hand, it's been already there...